### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by aulamber**

### CODE REVIEW

File: src/components/home/Home.js [hunk 1]
```diff
@@ -18,21 +22,35 @@ const styles = EStyleSheet.create({
...
 
+function mapDispatchToProps(dispatch) {
+  return {
+    actions: bindActionCreators(dummyActions, dispatch),
+  };
+}
+
 Home.propTypes = {
+  actions: PropTypes.shape().isRequired,
   navigation: PropTypes.shape().isRequired,
```
Comment (kimak) [inline src/components/home/Home.js]: missing details in shape, example:
```
 PropTypes.shape({
   navigation: PropTypes.func.isRequired,
}).isRequired,
```

File: src/reducers/navReducer.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+export default function navReducer(state, action) {
+  const newState = AppNavigator.router.getStateForAction(action, state);
+
+  if (action.type === types.PERSIST_REHYDRATE) {
+    return {
+      rehydrated: true,
+    };
+  }
+  return newState || state;
+}
```
Comment (kimak) [inline src/reducers/navReducer.js:13]: 👍

File: .eslintcache [hunk 3]
```diff
@@ -1 +1 @@
...
...,"warningCount":0}},"/Users/audreylambert/Documents/Poste-projets/TRIP/storybook/index.ios.js":{"size":380,"mtime":1488900798000,"hashOfConfig":"ia9ttb","results":{"filePath":"/Users/audreylambert/Documents/Poste-projets/TRIP/storybook/index.ios.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/audreylambert/Documents/Poste-projets/TRIP/storybook/stories/CenterView/index.js":{"size":308,"mtime":1488900798000,"hashOfConfig":"ia9ttb","results":{"filePath":"/Users/audreylambert/Documents/Poste-projets/TRIP/storybook/stories/CenterView/index.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/audreylambert/Documents/Poste-projets/TRIP/storybook/stories/CenterView/style.js":{"size":136,"mtime":1488900798000,"hashOfConfig":"ia9ttb","results":{"filePath":"/Users/audreylambert/Documents/Poste-projets/TRIP/storybook/stories/CenterView/style.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/audreylambert/Documents/Poste-projets/TRIP/storybook/stories/index.js":{"size":810,"mtime":1488900798000,"hashOfConfig":"ia9ttb","results":{"filePath":"/Users/audreylambert/Documents/Poste-projets/TRIP/storybook/stories/index.js","messages":[],"errorCount":0,"warningCount":0}}}
```
Comment (kimak) [inline .eslintcache]: i think we could remove this file and add it in gitignore

**Review by kimak — CHANGES_REQUESTED** [review]

File: .eslintcache (hunk 3 above)
Comment (aulamber) [reply .eslintcache]: Ok I'll see to it

File: src/components/home/Home.js (hunk 1 above)
Comment (aulamber) [reply src/components/home/Home.js]: Ok, I'll add it

File: .eslintcache (hunk 3 above)
Comment (aulamber) [reply .eslintcache]: What is strange is that the file has already been added to the gitignore.

**Review by kimak — APPROVED** [review]

**Pull request merged**